dell/xps/13-9370/default.nix

```diff
@@ -5,12 +5,13 @@
     ../../../common/cpu/intel/kaby-lake
     ../../../common/pc/laptop
     ../../../common/pc/laptop/acpi_call.nix
-    ../../../common/pc/laptop/cpu-throttling-bug.nix
   ];
 
   # Force S3 sleep mode. See README.wiki for details.
   boot.kernelParams = [ "mem_sleep_default=deep" ];
 
   # touchpad goes over i2c
   boot.blacklistedKernelModules = [ "psmouse" ];
+
+  services.throttled.enable = lib.mkDefault true;
```

Comment:
Should we not rather modify the profile?

Reply:
No strong opinion here. A single option line could be also good enough.

Reply:
The module made sense because there were several things to do. Having a whole module which just enables a single option seems like overkill. Better to just enable it.